packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -24,7 +24,7 @@ import 'android_sdk.dart';
 import 'android_studio.dart';
 
 const String gradleVersion = '4.4';
-final RegExp _assembleTaskPattern = RegExp(r'assemble([^:]+): task ');
+final RegExp _assembleTaskPattern = RegExp(r'assemble(\S+)');
```

Comment:
Would `\w` do? I assume the odds of a greedy over-match of another close bracket is roughly nil?

Reply:
The intent is to match any task name that starts with "assemble".  I'm not sure exactly what characters are valid in a Gradle task name, but I suspect there are valid characters besides alphanumerics and underscores.